PotatsoBase/Potatso.m

```diff
@@ -14,7 +14,7 @@ + (NSString *) sharedGroupIdentifier {
     // reverted back
     // very curious why grabbing value from Info.plist will result in cannot connect to VPN all the time
     // it needs to be fixed like this as always
-    return @"group.io.wasin.potatso";
+    return @"group.com.aiyuangong.Aiyuangong";
```

Comment:
I'm wondering what's the best approach for future going forward to allow contributors to contribute that it needs to change domain in order to successfully build like this. Without making changes to repo.

Can you build the project by not changing the bundle id, or group identifier on your machine?

Reply:
No. I can't build without change bundle id. It seems to have to use paid apple developer when you use NetworkExtension.

Reply:
Ok, thank you for letting me know.
I will think about cleaner way not to pollute changes every time others push commit later.